src/library_async.js

```diff
@@ -585,7 +603,7 @@ mergeInto(LibraryManager.library, {
         dbg('ASYNCIFY/FIBER: start rewind', asyncifyData, '(resuming fiber', newFiber, ')');
 #endif
         Asyncify.state = Asyncify.State.Rewinding;
-        _asyncify_start_rewind(asyncifyData);
+        _asyncify_start_rewind({{{ to64('asyncifyData') }}});
```

Comment:
Instead of doing this can you add `_asyncify_start_unwind` and `_asyncify_start_rewind` to this line in `emscripten.py`:

https://github.com/emscripten-core/emscripten/blob/b6670d7a79ebd219ada33f995ff61a4a8bacf441/emscripten.py#L855-L865

Reply:
That one did not help, as it only affects exports. So i've reverted to have {{{ to64 }}}

Reply:
But `_asyncify_start_unwind` and `_asyncify_start_unwind` are exports from the wasm file... perhaps because they are created by binaryen the are not visible at the time when this function runs...  in which case this approach seems OK for now, and I will try to amend later.

Reply:
OK.. I think maybe you tried adding them to the list with the `_` prefix?

The list in emscriptem.py contains unmangled symbols so these names should not have the `_` at the start when they appear there.

Reply:
Can you try again adding `asyncify_start_rewind`, `asyncify_start_rewind` (without leading underscores) to `create_pointer_conversion_wrappers`?